examples/others/imgfont/lv_example_imgfont_1.py

```diff
@@ -31,7 +31,6 @@ def get_imgfont_path(font, img_src, length, unicode, unicode_next, offset_y, use
 imgfont = lv.imgfont_create(80, get_imgfont_path, None)
 if imgfont == None:
     print("imgfont init error")
```

Comment:
```suggestion
    print("imgfont init error")
    exit()
```

Reply:
"exit" is not the right way to exit a python script.

Reply:
well I read some article about this, surely `sys.exit` is more correct.
```suggestion
    print("imgfont init error")
    import sys
    sys.exit(1)
```
Is OK for this?
exit and raise an error code 1

Reply:
Imports are usually grouped together at the beginning of the file.

Reply:
```suggestion
    print("imgfont init error")
    sys.exit(1)
```

Reply:
done